sos/report/plugins/openshift_ovn.py

```diff
@@ -32,6 +32,10 @@ def setup(self):
             "/var/lib/ovn-ic/etc/ovnnb_db.db",
             "/var/lib/ovn-ic/etc/ovnsb_db.db"
         ])
+        # Collect additional ovnkube logs for interconnect setup.
+        self.add_copy_spec([
+            "/var/lib/ovn-ic/etc/*log*"
```

Comment:
I'd also like to know what kind of logs are here, capturing *every* log file by default may not be ideal if there are other components logging here or if there is significant log rotation happening.

Reply:
hi @TurboTurtle reasonable questions. With the ovn0ic feature, we opted for having the libovsdb logs separate from the regular ovnkube controller. General details on this feature are here: 
More info on the logs are here: https://issues.redhat.com/browse/SDN-3507
CNO PR: https://github.com/openshift/cluster-network-operator/pull/1938
The logs are rotated via lumberjack and have the following values: 100Mb (uncompressed) and 5 rotated and compressed. With that, the worst case scenario will mean that the the SOS will have less than 500Mb.
How about we restrict the log files to use "libovsdb*log*" wildcard?
Here is an example output of what the files look like:
```
❯ POD=$(oc get pod -n openshift-ovn-kubernetes -l app=ovnkube-node -o jsonpath='{.items[0].metadata.name}')
ocn exec -it $POD -c ovnkube-controller -- bash -c "ls -lah /var/log/ovnkube/libovsdb*log*"
-rw-r-----. 1 root root  41K Aug 14 19:45 /var/log/ovnkube/libovsdb-2023-08-14T19-45-34.678.log.gz
-rw-r-----. 1 root root 660K Aug 14 23:38 /var/log/ovnkube/libovsdb.log
```

Reply:
What about collecting `/var/log/ovnkube/*log*` when `--all-logs` is specified, and `/var/log/ovnkube/*.log` otherwise. This follows the same kind of scenario for many of the plugins. This ensures that the size of sosreport is not as big if `--all-logs` is not specified

Reply:
I would say we want to have these files collected by default (since communicating the right flags that should be used is not that easy and may result in losing precious data), and we know this plugin is only active for openshift nodes, so this change shouldn't affect the majority of setups, wdyt?

Reply:
The prevailing approach in sos plugins is to capture `*.log` normally, and get `*log*` when either `--all-logs` or some plugin-specific option is used.

The `sizelimit` parameter here applies per item (in a list) handed to `add_copy_spec`, not the entire call or the individual matches for a regex (meaning a `sizelimit` setting of 600 is a bit heavy handed). So, another option would be having one entry for `*.log` and another for `*.log.gz` and setting sizelimit to 100 meaning by default we'd capture the active log and up to 100 MB of rotated logs. If something truly needs _all_ the rotated logs, then that's where `--all-logs` would come into play.

Would that be a reasonable middle ground?

Reply:
@npinaeva @TurboTurtle @arif-ali  , Thanks! Based on your concerns and suggestions, updated the log collection with commit b098433, Is that ok now ?